### ISSUE TITLE
[MultiVO] VO Specific FileCatalog administrator

### ISSUE THREAD

**Issue by andresailer** [DIRACGrid/DIRAC#4664]
I don't think it is possible at the moment to give some one the equivalent of the FileCatalogManagement property just for a `/<VO>` folder and below, e.g., to fix ownership of files and folders,  or is there?

**Comment by chaen** [comment]
I believe this was the original idea behind https://github.com/DIRACGrid/DIRAC/issues/4077 and https://github.com/DIRACGrid/DIRAC/pull/4230
Now before doing any more move, I am waiting for the doc.

**Comment by andresailer** [comment]
I don't think adding a static property is enough, it needs to be evaluated based on the path of the file
E.g. wherever this function is used?
https://github.com/DIRACGrid/DIRAC/blob/9a324b9406094b989fb64f90e2011ea301ea8f7d/DataManagementSystem/DB/FileCatalogComponents/SecurityManager/SecurityManagerBase.py#L81-L84

**Comment by chaen** [comment]
yes, this method also will need to be updated, depending on how/what one wants to check

**Comment by andresailer** [comment]
@atsareg Would be great to get your thoughts on this!

**Comment by andresailer** [comment]
Obviously not urgent, I'll just try something eventually.
